go/processor/floria/state_contract.go

```diff
@@ -83,24 +83,24 @@ func isStateContract(address tosca.Address) bool {
 	return address == StateContractAddress()
 }
 
-// handleStateContract is a reworked version of the original function from the Opera client.
+// runStateContract is a reworked version of the original function from the Opera client.
```

Comment:
nit: please aim for self-contained documentation, not assuming knowledge of third-party code from the perspective of the Tosca project.